packages/application-shell/src/utils/get-mc-api-url/get-mc-api-url.ts

```diff
@@ -0,0 +1,45 @@
+export interface ApplicationWindow extends Window {
+  app: {
+    servedByProxy?: string;
```

Comment:
```suggestion
    servedByProxy: string | boolean;
```

Reply:
Then we can't JSON.parse anymore cause... TS says it wants a string. Whatever, I'll change the check.

Reply:
Well you can do `JSON.parse(String(window.app.servedByProxy))`

Reply:
Come on! Then I'd rather be explicit. That's just type checker pleasing.

Reply:
The `JSON.parse` is not typed correctly in my opinion.

Anyway, how you changed it now it's better and more explicit. Let's keep that.

Reply:
I agree. Thanks your honor 😋.